team-policy.md

```diff
@@ -0,0 +1,144 @@
+# Style Team Policy Document
+
+This is a living document tracking the active policies of the style team. It is intended to fill a similar role to the books that many[^1] teams[^2] maintain[^3] independently[^4]. If in the future Rust establishes central linking/indexing/aggregating of policies, these will need to appear there.
+
+* **Original RFC**: https://rust-lang.github.io/rfcs/3309-style-team.html
+* **Charter**: https://github.com/rust-lang/fmt-rfcs/blob/master/charter.md
+
+## Style Guide Evolution
+
+The Rust style guide will generally match the latest version of the Rust style; the style team does not plan to maintain multiple branches of the style guide for different editions, in part because formatting for new constructs will apply to any edition supporting those constructs.
+
+Whenever possible, style decisions should be made before a new construct is stabilized. However, style decisions shall not be considered a blocker for stabilization.
+
```

Comment:
```suggestion

## Policy Review Process

To ensure that our policies remain up-to-date and effective, the style team implements a policy review process. This process involves setting review dates for all policies, documenting them alongside the policy itself, and including them as items in the backlog.

Review dates are selected based on the confidence level in a policy and its recent history of changes. If a policy has undergone recent modifications or is relatively new, a shorter review period is appropriate. This allows us to assess its impact and make any necessary adjustments promptly. On the other hand, if a policy has remained unchanged for an extended period and has proven to be effective, a longer review period can be chosen to minimize unnecessary energy expendeture.

By establishing review dates and incorporating them into our policy documentation and backlog, we ensure that policies are regularly evaluated and kept in line with the evolving needs of our team. This proactive approach help us keep our policies well-maintained and relevant.

```

Reply:
```suggestion

## Policy Review Process

To maintain the effectiveness and relevance of our policies, the style team implements a policy review process. The following guidelines outline how we conduct policy reviews:

1. Associated Review Dates: Each policy should have a designated review date, which is documented alongside the policy itself. These review dates are also added as items in the team's backlog for easy tracking and planning.
2. Consideration Factors: When selecting review dates for policies, we take several factors into account. These include:
  1. Confidence Level: If a policy has recently undergone changes or is relatively new, we opt for a shorter review period. This allows us to course correct quickly if needed.
  2. Stability and Effectiveness: Policies that have remained unchanged for a considerable period and have proven to be effective can have longer review periods. This prevents unnecessary time spent on policies that are working well.
3. Integration into Regular Meetings: Policy reviews are treated as regular agenda items during our team meetings. They are discussed and evaluated alongside other topics of importance.
4. Review Process: Policy reviews are conducted using a round-based approach. Each team member has the opportunity to provide feedback and raise any concerns or suggestions regarding the policy under review.
5. Regular Review Cadence: We review our policies at regular intervals to proactively identify any issues before they escalate into urgent problems. By conducting these reviews systematically, we ensure that our policies remain up to date and aligned with our evolving needs.

By adhering to this policy review process, we aim to maintain a high standard of quality and relevance in our style team policies.

```

Reply:
> 4. **Review Process**: Policy reviews are conducted using a round-based approach. Each team member has the opportunity to provide feedback and raise any concerns or suggestions regarding the policy under review.

I don't think this meta-policy on regular policy review is the right place to discuss how the discussion in the policy review should be facilitated. I also think we should be very careful about the degree of constraint we place on the facilitator for how they can most effectively facilitate discussion. In this context, I would prefer to state a goal, rather than a mechanism:

**Review Process**: Policy reviews should use a discussion approach that affords each team member the opportunity to provide feedback and raise any concerns or suggestions regarding the policy under review.

Reply:
Also, I'd like to include a point here about whether there's a default assumption of "keep the policy" or "drop the policy", since that seems like a major rationale for policy reviews:

**Default disposition**: In organizations without a policy review process, policies often remain in effect by default, until explicitly rescinded. The policy review process exists in part to reverse that default disposition: policies require regular review and must be renewed in order to remain in effect. (Such review and renewal still takes all of these other factors into account, such as the degree of previously established confidence in the policy, and its stability over time.)

Reply:
@joshtriplett I went ahead and updated the proposed wording to include your suggestions. I didn't remove the recommendation to use rounds, but I did change it to be clearly a recommendation and to first lead with the core goals of encouraging everyone to provide feedback and creating an environment where everyone is heard equally.

Reply:
@yaahc This solves most of my concern. My remaining concern with this new wording: we should add something to the prompt to invite people to raise issues with the policy other than just its efficacy at its intended purpose. (e.g. "this isn't the right purpose" or "this is causing other problems that aren't worth the tradeoff"...)

For instance, "Are there other issues with the policy that we should consider?"